k8s/prometheus/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,146 @@
 actions:
-- name: dummy test
+
+- name: Wait 60s to let Prometheus collect metrics
   bashTest:
     script: |-
-      exit 0
+      sleep 60
     expect:
       exitCode:
         equals: 0
+
+- name: Prometheus UI is available
+  bashTest:
```

Comment:
Can you use HttpTest instead (in all cases it's possible)? Example: https://github.com/GoogleCloudPlatform/click-to-deploy/pull/354/files#diff-db8d7657d07fa47983b136fa5ab73fe5R3